vault/identity_store.go

```diff
@@ -40,6 +40,43 @@ func (c *Core) IdentityStore() *IdentityStore {
 	return c.identityStore
 }
 
+func (c *Core) EntitiesInStorage(ctx context.Context) ([]*identity.Entity, error) {
```

Comment:
I think it'd be great to have a few tests in `identity_store_test.go` for this new function